steely/plugins/markov.py

```diff
@@ -14,12 +14,11 @@
 
 
 def main(bot, author_id, message, thread_id, thread_type, **kwargs):
-    if random.random() > CHANCE_OF_SEND:
-        return
+    return if random.random() > CHANCE_OF_SEND
```

Comment:
Hold on. I knew this was prominent in Ruby, but python can do this too???

Reply:
uhh kinda, I'm an idiot so what I did there was a ruby thing but [you *kinda can* do it in python](https://docs.python.org/2/reference/expressions.html#conditional-expressions)

Reply:
That's a shame, I would have adopted it.